update.sql

```diff
@@ -1,2 +1 @@
-use shopping;
```

Comment:
don't use a hardcoded database name in here - it would override the value we retrieved from config.php